api/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "eslint --fix src -c .eslintrc.json --ext js",
     "format": "prettier --write \"./**/*.{js,jsx,json,md}\"",
     "format:check": "prettier --check \"./**/*.{js,jsx,json,md}\"",
-    "test": "jest"
+    "test": "jest --setupFiles dotenv/config"
```

Comment:
what does this do? does it make it so that we don't need to add github secrets/change the api.yaml file for testing?

Reply:
I removed the dotenv code from the app.js in favor of preloading (specifying dotenv/config on the command line). All the other "start" scripts in the package.json have the preload arguments already, except for yarn test. I just added the preload args to yarn test, too.

So jest will use the .env file, but since the .env file doesn't exist on GitHub (just like test.env is empty on GitHub) we'll still have to add GitHub secrets.